src/integrations/admin/crawl-settings-integration.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use WPSEO_Admin_Asset_Manager;
+use WPSEO_Option_Tab;
+
+use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Presenters\Admin\Beta_Badge_Presenter;
+
+/**
+ * Crawl_Settings_Integration class
+ */
+class Crawl_Settings_Integration implements Integration_Interface {
+
+	/**
+	 * The admin asset manager.
+	 *
+	 * @var WPSEO_Admin_Asset_Manager
+	 */
+	private $admin_asset_manager;
+
+	/**
+	 * The options' helper.
+	 *
+	 * @var Options_Helper
+	 */
+	private $options_helper;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public static function get_conditionals() {
+		return [ Admin_Conditional::class ];
+	}
+
+	/**
+	 * Crawl_Settings_Integration constructor.
+	 *
+	 * @param WPSEO_Admin_Asset_Manager $admin_asset_manager The admin asset manager.
+	 * @param Options_Helper            $options_helper      The options helper.
+	 */
+	public function __construct(
+		WPSEO_Admin_Asset_Manager $admin_asset_manager,
+		Options_Helper $options_helper
+	) {
+		$this->admin_asset_manager = $admin_asset_manager;
```

Comment:
@pls78 I think that now the admin_asset_manager here is obsolete and can be removed

Reply:
Done!